test/browser_tests/page_objects/page_activity-log.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+function ActivityLog() {
+  this.get = function() {
+    browser.get( '/activity-log/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+  this.mainTitle = element( by.css( '[data-qa-hook="main-title"]' ) );
+
+  this.mainSummary = element( by.css( '[data-qa-hook="main-summary"]' ) );
+
+  this.searchFilter = element( by.css( '.js-post-filter' ) );
```

Comment:
You mean `[data-js-hook="filter"]`? I was trying to use the existing hooks where possible. We will need to replace all the existing js/qa hooks if everyone is comfortable with the approach.